HTML-Forms

digitekh2i opened the pull request:
In this project, I re-created the "intuit.com" sign up page.
-  Standard label form format
- Sign in and Create Account buttons

digitekh2i committed:
installed linters

digitekh2i committed:
added form page template structure in html file

digitekh2i committed:
style: added background color to the form page, positioned header element

digitekh2i committed:
added image folder and inserted icons and logo

digitekh2i committed:
added user input elements on the form page

digitekh2i committed:
Completed html page elements with text contents

digitekh2i committed:
styled the links and un-lists in css

digitekh2i committed:
placed the main form container at the center of the page

digitekh2i committed:
styled form label, submit botton and footer

digitekh2i committed:
re-styled company logo the 3 icons above the form

digitekh2i committed:
Added white background to form

digitekh2i committed:
form page final design

digitekh2i committed:
added screen image

tirthajyoti-ghosh reviewed (changes requested):
# Status: Changes required 🚨

## Hi @digitekh2i 🖐! Awesome project 🎉✨!

### However, we need to work on a couple of issues before we can move on.

- There are some Stylelint errors. Please fix them. From the next time, remember to fix any Styelint errors before requesting for a code review.

![image](https://user-images.githubusercontent.com/57726348/85223100-da4f5d00-b3dd-11ea-90de-b4c208eb0ec8.png)

- You have this PR from `gh-pages`. This branch is normally reserved for creating Github pages and not working on the features of a project. Remember to create a feature branch that has a descriptive name.

- Change the background color of the clone to match the original.

#### Clone

*background-color: #e9f0ea;*
![image](https://user-images.githubusercontent.com/57726348/85223302-b12fcc00-b3df-11ea-867c-69796ec7b57e.png)

#### Original

*background-color: #eceef1;*
![image](https://user-images.githubusercontent.com/57726348/85223284-965d5780-b3df-11ea-8737-dbf5ba1ab4bc.png)

- The phone input field must be of type `tel`, eg., `<input type="tel" ...`.

- The `captcha` part is missing.

#### Clone
![image](https://user-images.githubusercontent.com/57726348/85223754-cbb77480-b3e2-11ea-90bf-3bcb4d3e77ce.png)

#### Original
![Annotation 2020-06-21 171546](https://user-images.githubusercontent.com/57726348/85223772-015c5d80-b3e3-11ea-9fa9-fcb53eb399e1.jpg)
Remember to match its background color with the background color of the page.

- The lock icons are missing in the `sign-in` and `create account` buttons.

#### Clone
![Annotation 2020-06-21 171813](https://user-images.githubusercontent.com/57726348/85223840-83e51d00-b3e3-11ea-9d48-a6f980b096e3.jpg)

#### Original
![Annotation 2020-06-21 172001](https://user-images.githubusercontent.com/57726348/85223842-89426780-b3e3-11ea-8057-f606074576a9.jpg)

- There are some invalid CSS properties in your CSS file. Be sure to check the comments below.

tirthajyoti-ghosh commented on style.css:
```diff
@@ -0,0 +1,204 @@
+padding-top: 2px;
+padding-bottom: 30px;
+margin-top: 30px;
+margin-right: 150;
+margin-right: 150;
```
Don't forget to add units for your margin values, otherwise, they are deemed invalid by the browser.

tirthajyoti-ghosh reviewed (commented):
Apply these changes and request for another review.

Happy coding 👨‍💻

digitekh2i committed:
corrected CSS errors

digitekh2i committed:
changed page background

digitekh2i committed:
added google reCAPTCHA

digitekh2i committed:
added lock icons on the bottons

digitekh2i commented:
# changelog
- fixed CSS style errors
- changed the background color to fit the original page
- added reCAPTCHA section
- added lock icon to the buttons
- omitted CSS properties that are not in use

# I created a feature branch titled "dev-source" but while trying to update the pull request to compare master -> dev-source, as opposed to master -> gh-pages. I realized that it was doable. the available solution is to create another pull request that would compare master -> dev-source, but it is against the Microverse-Github Workflow. thus, that's why i am re-applying for code review with gh-pages again. Though I have taken corrections from all the feedback.

digitekh2i committed:
updated page screenshot

digitekh2i committed:
updated linter's html and css errors

elmejdki reviewed (approved):
# Great job @digitekh2i :clap: :clap: you have made a great job :tophat: 
### status: approved

now you can merge to master your project has been approved.

### Happy Coding :smiley:
